dev-server/documents/html/configurations.mustache

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Ignore configuration test</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <script type="application/json" class="js-hypothesis-config">
+      {
+        "theme": "clean",
+        "openSidebar": true
+      }
+    </script>
+    <script>
+      Object.defineProperty(window, 'hypothesisConfig', {
+        value: function () {
+          return {
+            openSidebar: false,
+            ignoreOtherConfiguration: true,
+          };
+        },
+      });
+    </script>
+  </head>
+  <body>
+    <h1>Ignore configuration test</h1>
+    <p>
+      `viahtml` sets the `ignoreOtherConfiguration` option on
+      `window.hypotesysConfig`. This option disregards other JSON hypothesis
```

Comment:
```suggestion
      `viahtml` sets the `ignoreOtherConfiguration` option in the result of
      `window.hypothesisConfig()`. This option disregards other JSON hypothesis
```

To be precise, `hypothesisConfig` is a function that returns an object, rather than an object itself as the previous wording might have suggested.